django_project/frontend/src/components/Home/index.tsx

```diff
@@ -0,0 +1,280 @@
+/**
+ * GeoSight is UNICEF's geospatial web-based business intelligence platform.
+ *
+ * Contact : geosight-no-reply@unicef.org
+ *
+ * .. note:: This program is free software; you can redistribute it and/or modify
+ *     it under the terms of the GNU Affero General Public License as published by
+ *     the Free Software Foundation; either version 3 of the License, or
+ *     (at your option) any later version.
+ *
+ * __author__ = 'zakki@kartoza.com'
+ * __date__ = '08/01/2025'
+ * __copyright__ = ('Copyright 2025, Unicef')
+ */
+
+import React, {Fragment, useEffect, useState, useRef} from 'react';
+import axios from "axios";
+import Grid from "@mui/material/Grid";
+import ImageIcon from '@mui/icons-material/Image';
+
+import {SearchInput} from "../Input/IconInput";
+import {MultipleSelectWithSearch, SelectWithSearch} from "../../components/Input/SelectWithSearch";
+import {SortAscIcon, SortDescIcon} from "../../components/Icons";
+
+import './style.scss';
+
+
+interface ProjectListProps {
+  baseUrl: string;
+  setIsLoading: (val: boolean) => void;
+}
+
+type Permission = {
+  list: boolean;
+  read: boolean;
+  edit: boolean;
+  share: boolean;
+  delete: boolean;
+}
+
+export interface GeoSightProject {
```

Comment:
Please put this here : https://github.com/unicef-drp/GeoSight-OS/blob/main/django_project/frontend/src/types.d.ts